<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/frame0/image_1.png</normaloff>
    <normalon>:/icons/frame0/image_1.png</normalon>:/icons/frame0/image_1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#navbar{
	background-color:#003a33;
}
#footer{
	background-color:#003a33;
}
#textheader{
	color:#ffffff;
}
#homeButton{
	background-color:#003a33;
	color:#ffffff;
	border: none;
}
#aboutButton{
	background-color:#003a33;
	color:#ffffff;
	border: none;
}
#container{
	background-color:#ffffff;
}
#containerVideo{
	background-color:#22706a;
	border-radius:20px;
	box-shadow: 20px 20px 20px 20px rgba(0, 0, 0, 0.5);
	border: 3px solid #aaa;   
}
#uploadButton{
	background-color:#22706a; 
	border:none;
}
#textcontainer{
	color:#ffffff;
}
#output_label{
	background-color:#003a33;
}
</string>
   </property>
   <widget class="QFrame" name="navbar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="icon" stdset="0">
     <string/>
    </property>
    <widget class="QLabel" name="textheader">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Crowd Detection App</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	color:#22706a;
}</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutButton">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	color:#22706a;
}</string>
     </property>
     <property name="text">
      <string>About</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="itenas_icon">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/icons/image_1.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icons/assets/image_1.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="footer">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>780</y>
      <width>1281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="text_footer">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#text_footer{
	color:#ffffff
}</string>
     </property>
     <property name="text">
      <string>2024 Group Activity 4, All Right Reserved</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icons/assets/image_2.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="text_footer_2">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#text_footer_2{
	color:#ffffff
}</string>
     </property>
     <property name="text">
      <string>Terms             Privacy           Contact us</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="container">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>110</y>
      <width>1281</width>
      <height>681</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="containerVideo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>1101</width>
       <height>541</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="textcontainer">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Track a Person by Crowd Detection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uploadButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover{
	color:#777777;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icons/assets/Image2.png</normaloff>:/icons/assets/Image2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
